token-transformer/src/main/java/com/networknt/rule/generic/token/TokenAction.java

```diff
@@ -92,8 +91,7 @@ private void parseActionValues() {
                     break;
                 }
                 case DESTINATION: {
-                    if (EnumUtils.isValidEnum(TokenSection.class, action.getValue()))
-                        this.tokenDestination = EnumUtils.getEnum(TokenSection.class, action.getValue());
+                    this.tokenDestination = TokenSection.valueOf(action.getValue());
```

Comment:
Could this call not hit "IllegalArgumentException"? Should we wrap in a try catch like this?

```
try {
     this.tokenDestination =  TokenSection.valueOf(action.getValue());
} catch(IllegalArgumentException e) {
    this.tokenDestination = TokenSection.NONE;
}
```